osbs/utils.py

```diff
@@ -387,6 +387,22 @@ def make_name_from_git(repo, branch, limit=53, separator='-', hash_size=5):
     return separator.join(filter(None, (repo, branch, hash_str)))
 
 
+def wrap_name_from_git(prefix, suffix, *args, **kwargs):
+    """
+    wraps the result of make_name_from_git in a suffix and postfix
+    adding separators for each.
+
+    see docstring for make_name_from_git for a full list of parameters
+    """
+    # 64 is maximum length allowed by OpenShift
+    # 2 is the number of dashes that will be added
+    prefix = ''.join(filter(VALID_BUILD_CONFIG_NAME_CHARS.match, list(prefix)))
+    suffix = ''.join(filter(VALID_BUILD_CONFIG_NAME_CHARS.match, list(suffix)))
```

Comment:
landscape.io suggests `''.join([VALID_BUILD_CONFIG_NAME_CHARS.match(ch) for ch in prefix])` etc
Is that easier to read? I tend to think what's already there is better.

Reply:
Agree, the existing one seems cleaner to me.